### DISCUSSION TITLE
Do i need SSL Certificate for github pages?

### DISCUSSION THREAD

**Discussion by nameismahipal** [community/community#22576] [Other Feature Feedback, Questions, & Ideas]
<p>I have set up Custom Domain at godaddy. While running website, I get as Insecure Content Blocked, and pages doesn’t load.</p>
<p>How to get this fixed ?</p>

**Comment by florincopil** [comment, marked as answer]
<p>GitHub is partnered free certificate authority “Let’s Encrypt”. Nowadays, custom domains on GitHub Pages are supporting for HTTPS. You have to update your DNS records with new IP address.</p>
<p>Once your DNS record is propagated, your site will be work accurately with HTTPS protocol. Now you should select “Enforce HTTPS” option under your repository’s settings to redirect your traffic from HTTP to HTTPS.</p>
<p>For more information - <a href="https://blog.github.com/2018-05-01-github-pages-custom-domains-https/" rel="nofollow noopener">https://blog.github.com/2018-05-01-github-pages-custom-domains-https/</a></p>

**Comment by johnhere6754** [comment]
GitHub is partnered with free certificate authority “Let's Encrypt”. Nowadays, custom domains on GitHub Pages are supported for HTTPS. You have to update your DNS records with a new IP address. Once your DNS record is propagated, your site for whatever niche it is [wash a car without a hose](https://washingears.com/how-to-wash-a-car-without-a-hose/) or ride a horse, etc all will work accurately with HTTPS protocol.
